django_tasks/backends/database/admin.py

```diff
@@ -3,9 +3,16 @@
 from django.contrib import admin
 from django.http import HttpRequest
 
+from django_tasks.task import ResultStatus
+
 from .models import DBTaskResult
 
 
+def retry(modeladmin: "DBTaskResultAdmin", request, queryset):
+    rows = queryset.update(status=ResultStatus.NEW)
```

Comment:
@RealOrangeOne I'm not sure if it's a good idea to reuse `DBTaskResult` rows; maybe we should be creating new rows instead. What do you think?

Reply:
For my use case (processing a bunch of documents), I find it useful to be able to retry failed tasks, until there are none left, fixing errors in the process. If we were to create new rows, then I'd have to stop the queue, retry the tasks, delete the old tasks, and then restart the queue.

Reply:
![image](https://github.com/user-attachments/assets/5f36e86b-f829-4477-8630-8a8ea68389e6)

Perhaps "Reenqueue" which updates the task, and a "Duplicate" which duplicates.

Reply:
Then we shall have both! I've updated the PR as you suggest, with a small test to make sure that duplicate tasks actually work.